src/cart/crt_context.c

```diff
@@ -228,6 +229,34 @@ crt_context_create(crt_context_t *crt_ctx)
 
 	D_RWLOCK_UNLOCK(&crt_gdata.cg_rwlock);
 
+	/** initialize sensors */
+	if (crt_gdata.cg_use_sensors) {
+		int	ret;
+
+		ret = d_tm_add_metric(&ctx->cc_timedout, D_TM_COUNTER,
+				      "Total number of timed out RPC requests",
+				      "", "net/%u/req_timeout", ctx->cc_idx);
```

Comment:
probably would be best to keep it as net/%s/%u/req_timeout, ctx->cc_provider, ctx->cc_idx so that it works automatically for multi-provider setup. 
Same for other stats

Reply:
fixed.